includes/admin/home/notices/class-sensei-home-notices-provider.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * File containing Sensei_Home_Notices_Provider class.
+ *
+ * @package sensei-lms
+ * @since   $$next-version$$
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Class that gathers and produces the local notices for Sensei Home.
+ */
+class Sensei_Home_Notices_Provider {
+	/**
+	 * The admin notices helper.
+	 *
+	 * @var Sensei_Admin_Notices
+	 */
+	private $admin_notices;
+
+	/**
+	 * Sensei_Home_News_Provider constructor.
+	 *
+	 * @param Sensei_Admin_Notices $admin_notices   The remote admin notices helper.
+	 */
+	public function __construct( $admin_notices = null ) {
+		$this->admin_notices = $admin_notices;
+	}
+
+	/**
+	 * Fallback for when we're in an environment that doesn't have `Sensei_Admin_Notices`.
+	 *
+	 * @return array
+	 */
+	private function local_only() : array {
+		/**
+		 * This filter is documented in `class-sensei-admin-notices.php`.
+		 */
+		$notices = apply_filters( 'sensei_admin_notices', [] );
+
+		return array_filter(
+			$notices,
+			function( $notice_key ) {
+				// We only care about home notices for now.
+				return strpos( $notice_key, Sensei_Home_Notices::HOME_NOTICE_KEY_PREFIX ) === 0;
+			},
+			ARRAY_FILTER_USE_KEY
+		);
+	}
+
+	/**
+	 * Returns all the information for the notices section.
+	 *
+	 * @param int $max_age The maximum age of the remote data.
+	 *
+	 * @return array
+	 */
+	public function get( $max_age = null ): array {
```

Comment:
$max_age is unused

Reply:
Removed in b6ba94c963731ba4fe2724032e306cb481739113